ilastik/applets/tracking/annotations/opAnnotations.py

```diff
@@ -293,7 +293,7 @@ def do_export(self, settings, selected_features, progress_slot, lane_index, file
             ott = partial(self.lookup_oid_for_tid, oid2tid)
             divs = [(value[1], ott(key, value[1]), key, ott(value[0][0], value[1] + 1), value[0][0],
                      ott(value[0][1], value[1] + 1), value[0][1])
-                    for key, value in sorted(divisions.iteritems(), key=itemgetter(0))]
+                    for key, value in sorted(iter(divisions.items()), key=itemgetter(0))]
```

Comment:
Automatic change by 2to3.
Answer is - principles laid down by PEP469
See here: http://legacy.python.org/dev/peps/pep-0469/#id9